tests/qos/qos_sai_base.py

```diff
@@ -2048,3 +2065,11 @@ def _skip_watermark_multi_DUT(
 
         yield
         return
+
+    @pytest.fixture(scope="function", autouse=False)
+    def skip_src_dst_different_asic(self, dutConfig):
+        if dutConfig['dutAsic'] != dutConfig['dstDutAsic']:
```

Comment:
hit KeyError: 'dstDutAsic'

Reply:
@XuChen-MSFT yes, this PR caused keyError issue, and PR https://github.com/sonic-net/sonic-mgmt/pull/10487 fixed it. 
So for double commit, I do double commit https://github.com/sonic-net/sonic-mgmt/pull/10300 and https://github.com/sonic-net/sonic-mgmt/pull/10487 together, to avoid this issue in other branches.